Helper/OmiseHelper.php

```diff
@@ -276,6 +276,36 @@ public function omiseAmountFormat($currency, $amount)
         return $amount;
     }
 
+    /**
+     * @param  string  $currency
+     * @param  integer $amount
+     *
+     * @return string
+     */
+    public function currencyAmountFormat($currency, $amount)
```

Comment:
What the purpose of this function? I'm not sure what `currencyAmountFormat` means?